### PR TITLE
Feature/production certificates

### PR DESCRIPTION
**Pull request by jeremyjams**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@Library('global-jenkins-library@2.1.1') _
+@Library('global-jenkins-library@feature/docker-build-options') _
```
Comment (jeremyjams) [inline Jenkinsfile:1]: To update before merge

File: cloud-computing/gramine/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # FIXME: use tagged version when released
-FROM docker-regis.iex.ec/iexec-graphene-base:0.7.0
+FROM docker-regis.iex.ec/iexec-graphene-base:feature-bad30168
```
Comment (jeremyjams) [inline cloud-computing/gramine/Dockerfile:2]: To update before merge

File: Jenkinsfile [hunk 3]
```diff
@@ -11,14 +11,32 @@ nativeImage = buildSimpleDocker_v3(
...
+        productionImageName = buildSimpleDocker_v3(
+            buildInfo: buildInfo,
+            dockerfileDir: dockerfileDir,
+            buildContext: baseDir,
+            dockerImageRepositoryName: dockerImageRepositoryName,
+            visibility: visibility
+        )
+    }
+    stage('Build Gramine test CA Gramine image') {
+        buildInfo.imageTag = buildInfo.imageTag + '-test-ca'
```
Comment (jbern0rd) [inline Jenkinsfile:29]:
```suggestion
        buildInfo.imageTag += '-test-ca'
```

File: Jenkinsfile [hunk 4]
```diff
@@ -11,14 +11,32 @@ nativeImage = buildSimpleDocker_v3(
...
+            dockerfileDir: dockerfileDir,
+            dockerfileFilename: 'Dockerfile.test-ca',
+            dockerBuildOptions: '--build-arg BASE_IMAGE=' + productionImageName,
+            dockerImageRepositoryName: dockerImageRepositoryName,
+            visibility: visibility
+        )
+    }
+}
 
 sconeBuildUnlocked(
```
Comment (jbern0rd) [inline Jenkinsfile:41]: `buildInfo.imageTag -= '-test-ca'`

Comment (jbern0rd) [reply Jenkinsfile:41]: `def gramineSuffix = '-test-ca'` to avoid errors ?

**Review by mcornaton — APPROVED** [review]
Looks great, thanks!

File: cloud-computing/gramine/Dockerfile [hunk 5]
```diff
@@ -1,5 +1,5 @@
 # FIXME: use tagged version when released
-FROM docker-regis.iex.ec/iexec-graphene-base:0.7.0
+FROM docker-regis.iex.ec/iexec-graphene-base:0.8.0
 
 RUN  apt-get install -y python3 \
```
Comment (jbern0rd) [inline cloud-computing/gramine/Dockerfile:4]: Would it be possible to clean apt cache here before merge ?

Comment (jbern0rd) [reply cloud-computing/gramine/Dockerfile]: Thanks

**Review by jbern0rd — APPROVED** [review]

File: cloud-computing/gramine/Dockerfile (hunk 5 above)
Comment (jeremyjams) [reply cloud-computing/gramine/Dockerfile]: You'r welcome, thanks :)

**Pull request merged**